message/message.go

```diff
@@ -0,0 +1,113 @@
+package message
+
+import (
+	"time"
+
+	"github.com/apache/arrow/go/v13/arrow"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+)
+
+type Message interface {
+	GetTable() *schema.Table
+}
+
+type MigrateTable struct {
+	Table *schema.Table
+}
+
+func (m MigrateTable) GetTable() *schema.Table {
+	return m.Table
+}
+
+type Insert struct {
+	Record arrow.Record
+	Upsert bool
```

Comment:
I remember something in Slack about removing the `Upsert`, but it seems we do use it